packages/lib-react-components/src/Loader/Loader.js

```diff
@@ -0,0 +1,68 @@
+import { string } from 'prop-types'
+import { Box } from 'grommet'
+import { useTranslation } from '../translations/i18n'
+import styled from 'styled-components'
+
+const StyledBar = styled(Box)`
+  @keyframes bounce-height {
+    0% {
+      height: 20%;
+    }
+
+    10% {
+      height: 100%;
+    }
+
+    100% {
+      height 20%;
+    }
+  }
+
+  animation: bounce-height 1s linear 0s infinite;
+  height: 20%;
+  width: 10%;
+  border-radius: 4px;
+  background: ${props => props.theme.global.colors.brand};
+
+  &.delayed-one {
+    animation: bounce-height 1s linear 200ms infinite;
+  }
+
+  &.delayed-two {
+    animation: bounce-height 1s linear 500ms infinite;
+  }
+`
+
+function Loader({
+  height = 'xxsmall',
+  loadingMessage = '',
+  width = 'xxsmall'
+}) {
+  const { t } = useTranslation()
+
+  return (
+    <Box
+      a11yTitle={loadingMessage || t('Loader.loading')}
+      role='status'
```

Comment:
At the moment, this status live region will be removed from the page when the loader disappears. I think it would be better to keep the live region permanently on the page, but change the content as the loader appears and disappears. Assistive technology will only announce the content of a live region when it changes.

See also Patrick Lauke's recent post: [Why are my live regions not working?](https://tetralogical.com/blog/2024/05/01/why-are-my-live-regions-not-working/) In particular, this status won't be announced because its content is static.

Reply:
I'd also consider allowing the container to passing a custom loading message eg. 'Workflows loading/Workflows loaded' for the project workflows on the home page. Otherwise, the generic 'loading/loaded' message doesn't tell me which part of the page is loading.

Reply:
Both helpful comments, but this PR is purely to move development of the new homepage designs forward. Further changes to Loader handling can happen at a future date and in a different scope.

Reply:
Sure, but it might be worth documenting in an issue that the status code here doesn’t work.